.github/workflows/package_installers.yml

```diff
@@ -133,6 +136,19 @@ jobs:
            --app-image "${{ env.MACOS_TARGET_PATH }}" \
            --dest ${{ env.MACOS_TARGET_DEST_PATH }}
 
+      - name: Package cli app jar with Gradle
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: shadowJar
+
+      - name: Persist cli app jar
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        uses: actions/upload-artifact@v3
```

Comment:
Please update all upload-artifact versions to `4`.

Reply:
updated all nodejs 16 deprecated actions: https://github.com/MobilityData/gtfs-validator/actions/runs/8071666378